Automattic/jetpack
# Admin Page: only display extra cards to users that will be interested in it

> **Warning**
> Do not merge until #33263 has been merged.

## Proposed changes:

- The Apps card displays information about stats, so we can only display it to users who have access to stats.
- Typically only admins contact us for Jetpack support, so I think we can choose to only display that card to admins.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

I would recommend testing once you've done the whole setup for #33263.

Once you have that setup, access Jetpack > Dashboard with your admin account, and in another browser with your subscriber account. Ensure you only see the cards that matter.
